core/notifications/src/notification_event/marketing_notification_triggered.rs

```diff
@@ -12,7 +12,7 @@ pub struct MarketingNotificationTriggered {
     pub should_send_push: bool,
     pub should_add_to_history: bool,
     pub should_add_to_bulletin: bool,
-    pub deep_link: Option<DeepLink>,
+    pub deep_link: Option<DeepLink>, // This is no longer used but kept for deserialization of old events
```

Comment:
can use ```#[serde(skip)]```
https://serde.rs/field-attrs.html#skip

Reply:
Can just delete - serde will ignore superfluous attributes by default.

Reply:
Removed it